lib/App/Frontend.php

```diff
@@ -34,6 +34,9 @@ class App_Frontend extends App_Web
      */
     public $layout = null;
 
+    /** @var App_Frontend */
+    public $app;
```

Comment:
yes that's because of type change. We need such things in sub-classes to correctly type-hint parent class properties.